compliance_checker/cf/cf.py

```diff
@@ -1910,7 +1904,7 @@ def check_calendar(self, ds):
                                     valid_calendar,
                                     '§4.4.1 Time and calendar',
                                     reasoning)
-                ret_val.append(result)
+                    ret_val.append(result)
```

Comment:
According to CF, `calendar` is actually optional. So the only checks that it recommends is that if it exists, that it be a valid value according to the list in 4.4.1.

I'm ok with returning results that pass if the attribute doesn't exist.